src/Webapi/Webapi__ReadableStream.re

```diff
@@ -0,0 +1,30 @@
+[@text "@since 0.19.0"];
+
+module type Reader = {
+  type t;
+  type closed;
+
+  [@bs.send] external closed: t => Js.Promise.t(closed) = "";
+  [@bs.send] external cancel: t => Js.Promise.t(unit) = "";
+  [@bs.send.pipe: t] external cancelWith: string => Js.Promise.t(string) = "cancel";
+  [@bs.send] external releaseLock: t => unit = "";
+};
+
+module rec DefaultReader: {
+  include Reader;
+  [@bs.send] external read: t => Js.Promise.t(Fetch__Iterator.Next.t(string)) = "";
```

Comment:
Where is `Fetch__Iterator` defined? I am getting compile error saying this module cannot be found. @yawaramin

Reply:
Nvm, I trying to upgrade my project to bs-platform 0.8.2, I found where it is defined. bs-fetch 👍

Reply:
Thanks for reminding me about this, I've added a comment to the `Webapi` documentation.